packages/ui/src/components/molecules/MessageBox/Message.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+
```

Comment:
pass `width` and `widthProps` as {...props}

Refer to this file [here](https://github.com/Cypherock/cypherock-cysync/blob/eae819a5a1f2ea2b89511628e9c7a617ffbc07b9/packages/ui/src/components/molecules/OneInMany.tsx)